Adding settings recipe for graphql (#610)
vtex-apps/io-documentation: docs/en/Recipes/development/developing-service-configuration-apps.md

3. Once successfully cloned, go to the local app directory. Use the `cd service-example` command;
4. Open the app using your code editor;

> 5. Now, to declare that your app can receive configurations through requests, check [this section](#if-your-app-has-a-node-service) if your app has a Node service and [this section](#if-you-are-developing-a-graphql-app) if you are developing a GraphQL app.

Reviewer: ```suggestion
5. Now, to declare that your app can receive configurations through requests, check the sections "If your app has a node service" or "If you are developing a GraphQL app", according to your scenario.
```

  },
  ...
}
```

### If your app has `grapqhl/schema.graphql` file

> If you are developing a GraphQL app, you may need to add a directive to all of the queries that can receive configurations. This is pretty easy. If you'd add this directive to [the GraphQL `vtex init` app](https://github.com/vtex-apps/graphql-example), in the query `book`, the result would be:

Reviewer: ```suggestion
If you are developing a GraphQL app, you may need to add a directive to all of the queries that can receive configurations. 

For example, consider our [graphql-example](https://github.com/vtex-apps/graphql-example) app (you can play with it if you run `vtex init` in a development workspace and choose `graphql-example`). In this app's root directory, you'll see the following file `grapqhl/schema.graphql`. Now, if you open it and add the `@settings` directive to the query `book`, you'll have something like:

```

Reviewer: "Why would I need directives for queries that can receive configurations? Which directives could I want to use?" - I think this could be a bit more explored. 

Also, for the example, what does the inclusion of the @settings directive do?

We don't recommend the use of this kind of sentence "**This is pretty easy**". Cause if the person is not able to do it for any reason, she can get pissed off D:

Author: Makes sense. Directives is just a way of configuring how the query will be performed. I'll add some more details to the text! Thanks for the feedback!

  ...
}

> 

Reviewer 2: can you terminate the ``` block here?

### If your app has `grapqhl/schema.graphql` file
If you are developing a GraphQL app, you may need to add a directive to all of the queries that can receive configurations. This is pretty easy, this is the directive:
```
@settings(settingsType: "workspace")
```

> If you'd add this directive to the GraphQL `vtex init` app, in the query `book`, the result would be:

Reviewer 2: can you link to the repo of the template here?

``` diff
type Query {
-  book(id: ID!): Book
+  book(id: ID!): Book @settings(settingsType: "workspace")
}

> +@settings(settingsType: "workspace")

Reviewer 2: this isn't needed
